create/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -1,7 +1,7 @@
 :_mod-docs-content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
 [id="remote-pipelines-tasks-resolvers"]
-= Specifying remote pipelines and tasks using resolvers
+= Specifying remote pipelines, tasks, and StepActions using resolvers
```

Comment:
IMHO we should keep the same style for all resources here, i.e. `step actions` instead of `StepAction`

Reply:
doing this, yes